app/controllers/concerns/unconfirmed_user_concern.rb

```diff
@@ -62,7 +62,7 @@ def process_valid_confirmation_token
   def process_unsuccessful_confirmation
     @confirmation_token = params[:confirmation_token]
     flash[:error] = unsuccessful_confirmation_error
-    redirect_to sign_up_email_resend_url(request_id: params[:_request_id])
+    redirect_to sign_up_register_path(request_id: params[:_request_id])
```

Comment:
I think at one point in time we preferred `_url` helpers to `_path` helpers but I forget if it matters?

Reply:
I recall a similar discussion where it ended up that it doesn't matter much for redirects since Rails will end up converting it to a fully-qualified URL regardless. I don't really hold a strong opinion, just a general skepticism of fully-qualified URLs (ensuring that the host is what we expect it to be) and a little bit more manageable as a smaller fragment of data if we're testing against it, etc.

Reply:
But also in favor of being consistent for consistency's sake, if we have a common convention. I can update.

Reply:
nah it just jumped out to me in the diff, I wouldn't worry about it. I'll see if I can dig up the old reasoning

Reply:
Too late, I updated in 6c93fe3 😄